gradle.properties

```diff
@@ -0,0 +1,12 @@
+org.gradle.caching=true
+group=io.ballerina.stdlib
+version=4.0.0-SNAPSHOT
+
+checkstylePluginVersion=10.12.0
+spotbugsPluginVersion=5.0.14
+shadowJarPluginVersion=8.1.1
+downloadPluginVersion=5.4.0
+releasePluginVersion=2.8.0
+testngVersion=7.6.1
+eclipseLsp4jVersion=0.12.0
+ballerinaGradlePluginVersion=2.1.5

```

Comment:
Let's use the latest version.

Reply:
Fixed in [53cae63](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/53cae6393e24c09f4b87e8f27a6a6a4b2c6c4a15)